Project.toml

```diff
@@ -5,15 +5,15 @@ version = "0.4.1"
 
 [deps]
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
-GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
 GeometryBasics = "5c1252a2-5f33-56bf-86c9-59e7332b4326"
+Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
 Observables = "510215fc-4207-5dde-b226-833fc4488ee2"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 julia = "1.6"
 FillArrays = "1.0"
-GLMakie = "0.8"
+Makie = "0.19"
```

Comment:
Should we update to `0.20`?

Reply:
Should I bump the version number for InteractiveViz too while I'm at it (to `v0.5.0`)?

Reply:
Yes please